src/components/sbb-header-action/sbb-header-action.scss

```diff
@@ -7,34 +7,30 @@ $breakpoints: 'zero', 'micro', 'small', 'medium', 'large', 'wide', 'ultra';
 
 :host {
   --sbb-header-action-text-display: none;
-  --sbb-header-action-icon-margin: 0;
   --sbb-header-action-padding-inline: 0;
   --sbb-header-action-background-color: transparent;
   --sbb-header-action-min-height: var(--sbb-size-button-m-min-height);
   --sbb-header-action-min-width: var(--sbb-header-action-min-height);
 
-  display: inline-block;
-}
+  // Variable is assigned to property from sbb-header if it is first element
+  // in header and needs left offset correction.
+  // Move it left by distance between button's left side and the start of the icon.
+  // As result, the icon should be aligned with the left side of the page wrapper.
+  --sbb-header-first-item-margin-inline-start: calc(
+    -0.5 * (var(--sbb-header-action-min-width) - var(--sbb-size-icon-ui-small))
+  );
 
-:host(:first-child) {
-  margin-left: calc(var(--sbb-spacing-fixed-2x) * (-1));
-  @include mq($to: small) {
-    margin-left: calc(var(--sbb-spacing-fixed-3x) * (-1));
-  }
+  display: inline-block;
 }
 
 @each $breakpoint in $breakpoints {
   :host([expand-from='#{$breakpoint}']) {
     @include mq($from: $breakpoint) {
       --sbb-header-action-text-display: block;
-      --sbb-header-action-icon-margin: var(--sbb-spacing-fixed-2x);
       --sbb-header-action-padding-inline: var(--sbb-spacing-fixed-5x);
-      --sbb-header-action-min-width: inherit;
-    }
-  }
-  :host([expand-from='#{$breakpoint}']:first-child) {
-    @include mq($from: $breakpoint) {
-      margin-left: calc(var(--sbb-spacing-fixed-5x) * (-1));
+      --sbb-header-first-item-margin-inline-start: calc(
+        var(--sbb-header-action-padding-inline) * (-1)
```

Comment:
`(-1)` seems a bit weird? Is it auto-formatted like this?

Reply:
I reformatted the line without brackets.